deploy/olm/openshift/0.7.0/dynatrace-monitoring.v0.7.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,439 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - security.openshift.io
+          resourceNames:
+          - privileged
+          - host
+          resources:
+          - securitycontextconstraints
+          verbs:
+          - use
+        serviceAccountName: dynatrace-oneagent
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - nodes
+          verbs:
+          - get
+          - list
+          - watch
+        serviceAccountName: dynatrace-oneagent-operator
+      deployments:
+      - name: dynatrace-oneagent-operator
+        spec:
+          replicas: 1
+          revisionHistoryLimit: 1
+          selector:
+            matchLabels:
+              name: dynatrace-oneagent-operator
+          strategy:
+            type: Recreate
+          template:
+            metadata:
+              labels:
+                dynatrace: operator
+                name: dynatrace-oneagent-operator
+                operator: oneagent
+            spec:
+              affinity:
+                nodeAffinity:
+                  requiredDuringSchedulingIgnoredDuringExecution:
+                    nodeSelectorTerms:
+                    - matchExpressions:
+                      - key: beta.kubernetes.io/arch
+                        operator: In
+                        values:
+                        - amd64
+                        - arm64
+                      - key: beta.kubernetes.io/os
+                        operator: In
+                        values:
+                        - linux
+                    - matchExpressions:
+                      - key: kubernetes.io/arch
+                        operator: In
+                        values:
+                        - amd64
+                        - arm64
+                      - key: kubernetes.io/os
+                        operator: In
+                        values:
+                        - linux
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: RELATED_IMAGE_DYNATRACE_ONEAGENT
```

Comment:
Remove it from here since 0.7.0 was released and doesn't have this. Just add it to the openshift/deployment.yaml deployment. It should be included in the v0.7.1 CSV once it gets generated.